app/scripts/modules/titus/src/pipeline/stages/runJob/TitusRunJobStageConfig.tsx

```diff
@@ -218,7 +236,69 @@ export class TitusRunJobStageConfig extends React.Component<IStageConfigProps, I
           onChange={this.dockerChanged}
           deferInitialization={stage.deferredInitialization}
         />
-
+        <h4>
+          <b>
```

Comment:
probably don't need the <b> tags

Reply:
left the ` <b>` tag to make it look consistent with deploy UI , let me know if you think we need to remove that.